nightingale_dispatcher/src/nightingale_dispatcher/main.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import rospy
+
+import queue
+from nightingale_msgs.srv import AddTaskDispatcher, CancelTaskDispatcher
+from nightingale_dispatcher.subtask import Subtask
+from nightingale_dispatcher.navigate_subtask import NavigateSubtask
+
+
+class DispatcherNode:
```

Comment:
Are there ways to pre-empt/pause tasks?  I'm thinking because for the hand bin tasks we will need user input which will require waiting on the tablet input and calling of the "interface call" task. Unless the connection to the interface is not to be in a task?

Reply:
@FelixKHWong , based on our more recent discussions, is this conversation resolved?

Reply:
I think with how the new 'tasks' are structured by specifying the different actions within the function we are able to do this. This is resolved